docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,91 +24,248 @@ The following table shows how risk levels are applied to a host, based on the no
 
 |==============================================
 
+[[enable-host-risk-score]]
 [discrete]
-=== Deploy host risk score
+=== Enable host risk score
 
-NOTE: To enable the host risk score feature, you must have alerts in your environment. If you have none, the *Enable via Dev Tools* button will be greyed out.
+NOTE: To enable the host risk score feature, you must have alerts in your environment. If you previously enabled host risk score and are upgrading the {stack} to 8.5 or newer, refer to <<upgrade-host-risk-score>> in this topic. 
 
-To deploy the host risk score framework in your environment:
+You can enable host risk score from the following places in the  {security-app}:
```

Comment:
@angorayc @SourinPaul Do we want to have multiple places for users to enable host risk score? I was under the impression that we were switching over to only enabling via the centralized EA page starting 8.6.

Reply:
We share the same UI of enabling / upgrading the module across all the pages, so it's true that in 8.5 users can enable risk score from multiple pages.